Cart/cart.css

```diff
@@ -95,3 +96,30 @@
     background-color: var(--color-primary);
     width: 100%;
 }
+
+@media screen and (max-width: 500px){
+    .horizontal-img{
+        width: 100px;
+    }
+    .minus,.add,.qty-count{
```

Comment:
You can give space here after each class like `.minus, .add, .qty-count`

Reply:
Yes sure !!